advocacy_docs/edb-postgres-ai/ai-accelerator/reference/index.mdx

```diff
@@ -21,6 +21,8 @@ navigation:
 * [aidb.create_model](models#aidbcreate_model)
 * [aidb.get_model](models#aidbget_model)
 * [aidb.delete_model](models#aidbdelete_model)
+* [aidb.get_hcp_models](models#aidbget_hcp_model)
```

Comment:
The headline is called "models" hence why the link isn't working. Not sure if you want to change the headline or the link, here is a fix for the link. 
```suggestion
* [aidb.get_hcp_models](models#aidbget_hcp_models)
```

Reply:
oh, I guess the function name is models, so it makes sense to alter it here.